promql/promqltest/README.md

```diff
@@ -73,8 +75,8 @@ When loading a batch of classic histogram float series, you can optionally appen
 
 ## `eval` command
 
-`eval` runs a query against the test environment and asserts that the result is as expected.
-It requires the query to succeed without any (info or warn) annotations.
+`eval` runs a query against the test environment and asserts that the result is as expected. 
+It requires the query to succeed without any failures.
```

Comment:
Unless you provide an `expect fail` line?

Reply:
Let's also explicitly mention that the current behavior is different from the previous one where a simple `eval` would require no (info or warn) annotations and that you need to write `expect no_info` or `expect no_warn` now to create the same behavior.

Reply:
Done!